httpserver/service_node.cpp

```diff
@@ -194,12 +153,6 @@ ServiceNode::ServiceNode(boost::asio::io_context& ioc,
 
     ping_peers_tick();
 
-    worker_thread_ = std::thread([this]() { worker_ioc_.run(); });
-    boost::asio::post(worker_ioc_, [this]() {
-        pow_difficulty_timer_tick(std::bind(
-            &ServiceNode::set_difficulty_history, this, std::placeholders::_1));
-    });
-
     boost::asio::post(worker_ioc_,
                       [this]() { this->check_version_timer_tick(); });
```

Comment:
Without a thread running `worker_ioc_` this isn't going to work -- but leave it, I have an incoming PR that rips out the version check entirely (because no one even noticed that the DNS entry wasn't updated since 1.0.4) and I'll just amend it to take out the worker_ioc_ as well.